src/surfing_penguin/db_interface/SurveyFunctions.py

```diff
@@ -36,3 +36,30 @@ def new_question(survey, data):
     question = Question(data["title"], data["content"], survey)
     session.add(question)
     session.commit()
+
+
+def new_answerlist(data):
```

Comment:
It would be great if there are some comments about this function and the structure of data.